src/utilities/mouse_utils.py

```diff
@@ -1,40 +1,73 @@
 import pyautogui as pag
 import numpy as np
+import random as rd
+from typing import Callable, Union
 
 
-class MouseUtils():
+class MouseUtils:
 
-    def move_to(self, point: tuple, duration=0.3, variance=0):
+    @staticmethod
+    def move_to(point: tuple, duration: float = 0.3, duration_variance: float = 0, time_variance: float = 0,
```

Comment:
The `variance` param (now `duration_variance`) is actually the variance in the _destination_ position of the mouse, rather than the time it takes to get there. So I think any word that implies destination works best: 'dest', 'destination', etc. I have a feeling that's what you were going for but just mixed them up - no worries!

Reply:
Ahhh I totally got them mixed up! Pushing a fix now!

Reply:
Also, one quick note before you do, the type hint should be an integer 😃

Reply:
good catch, I changed the lint in another push! (also I forgot to make the fix in rel_to)